criu/net.c

```diff
@@ -339,6 +352,72 @@ static int ipv6_conf_op(char *tgt, SysctlEntry **conf, int n, int op, SysctlEntr
 			devconfs6, def_conf);
 }
 
+static int unix_conf_op(SysctlEntry ***rconf, size_t *n, int op)
+{
+	int i, ret = -1, flags = op == CTL_READ ? CTL_FLAGS_OPTIONAL : 0;
```

Comment:
could we simplify this line, for example, with something like:
```C
int flags = 0;
....
if (opts.weak_sysctls || op == CTL_READ)
	flags = CTL_FLAGS_OPTIONAL;
```

Reply:
sure. Thanks ;)